### PR TITLE
Dean/new better visuals

### PR DESCRIPTION
**Pull request by dijour**
Added a new battery class, which has a decrement/increment function to change the Sprite texture from 0-3 battery bars. I added a minecart that shakes at a regular interval, and I moved the 3 hearts further up the screen so they don't collide with left flying bats.

### CODE REVIEW

File: ddr/ddr/Models/GameEntities/BatteryBar.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    }
+  }
+  
+  func decrement() {
+    if charge < 3 {
+      charge += 1;
+      self.batteryTexture = SKTexture(imageNamed: "battery\(charge)")
+    }
+  }
+}
```
Comment (mjkern) [inline ddr/ddr/Models/GameEntities/BatteryBar.swift:37]: Does the battery provide any visual indication of how charged it is?

**Comment by dijour** [comment]
Yes - that's what the change to batteryTexture does. We should tie this to
the flashlight logic though.

**Comment by mjkern** [comment]
Yeah, it would be great if you could do that

**Review by Aklaran — APPROVED** [review]
Once we merge this, I will add the battery as an optional param to Flashlight and hook into the BatteryBar class.

**Pull request merged**